src/components/landing.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import lnd from '../images/Lion.png';
+
+const Landing = () => (
+  <div className="Landing">
+    <img src={lnd} alt="Gif" />
+    <div className="LandingDetails">
+      <h2>Lions</h2>
+      <p>The king of the jungle</p>
+    </div>
+  </div>
+);
+export default Landing;
```

Comment:
- [ ] Your landing section on the homepage is not as descriptive of the elements on the homepage as what we have on the design. In the design, **EUROPE** and a map of it describes the content which includes the Czech Republic, Italy and so on which are typically countries in the European continent. In your implementation, **LION** and its image, does not describe the homepage contents. Kindly resolve this.

  Your Implementation | Given Design
  :---:|:---:
  ![nemwel-landing](https://user-images.githubusercontent.com/36542986/164276896-043c99d4-1b0f-45e6-942e-3f70033c7955.jpg) | ![akumu-descriptive](https://user-images.githubusercontent.com/36542986/164276575-6d6fce0f-cde9-43ab-97c8-3384ac5c1f74.jpg)